yaksh/models.py

```diff
@@ -335,14 +389,53 @@ def is_expired(self):
 
     def has_prerequisite(self):
         return True if self.prerequisite else False
-
+    
     def __unicode__(self):
         desc = self.description or 'Quiz'
         return '%s: on %s for %d minutes' % (desc, self.start_date_time,
                                              self.duration)
 
 
 ###############################################################################
+class QuestionPaperManager(models.Manager):
+
+    def _copy_original_questionpaper(self, original_quiz_id):
+        trial_questionpaper = self.get(quiz_id = original_quiz_id)
+        trial_questions = {"fixed_questions": trial_questionpaper\
+                                                .fixed_questions.all(),
+                            "random_questions": trial_questionpaper\
+                                                .random_questions.all()
+                            }
+        trial_questionpaper.pk = None
+        trial_questionpaper.save()
+        return trial_questionpaper, trial_questions
+
+    def _create_trial_questionpaper(self, trial_quiz):
+        trial_questionpaper=self.create(quiz=trial_quiz,
+                                        total_marks=10,
+                                        )
+        return trial_questionpaper
+
+    def add_details_trial_questionpaper(self, trial_quiz,
```

Comment:
Can't this be `create_trial_paper` with suitable arguments with a proper docstring?